app/not-found.tsx

```diff
@@ -0,0 +1,12 @@
+import { Container } from '../components';
```

Comment:
Changed the absolute imports to relative imports, because we use the Upleveled ESLint tsconfig.json and don't have paths defined

    "paths": {
      "@components": [
        "./components"
      ],
      "@utils/*": [
        "./utils/*"
      ],
    }